docs/general/attributes.md

```diff
@@ -320,7 +320,7 @@ These attributes may be used for any operation with an authenticated and/or auth
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
 | `enduser.id` | string | Username or client_id extracted from the access token or [Authorization](https://tools.ietf.org/html/rfc7235#section-4.2) header in the inbound request from outside the system. | `username` | Recommended |
-| `enduser.role` | string | Actual/assumed role the client is making the request under extracted from token or application security context. | `admin` | Recommended |
+| `enduser.role` | string | Actual/assumed role(s) the client is making the request under extracted from token or application security context. | `admin, superadmin` | Recommended |
```

Comment:
another option is to change it to `enduser.roles` and define it as `string[]`

Reply:
Yeah that'd be clearer.  `enduser.scope` could also be pluralized and changed to `string[]`

Do you know off-hand if this change can/should have a transformation in `schema-next.yaml`? I don't see other examples of a rename that also goes from `string` -> `string[]`, and didn't see the [schema documentation](https://opentelemetry.io/docs/specs/otel/schemas/) mention if/how data type changes are handled

Reply:
I found the [schema transformation processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/schemaprocessor) and will take a look at how this scenario's handled

Reply:
the processor just renames the attribute - no transformation to the value's done

lmk if the breaking change to string[] makes sense